.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Let's not include a gitignore in the PR since its possible to have git exclude these files locally https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally